### ISSUE TITLE
Add term sclerotic foci of the femur

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10469]
**Preferred term label:**
Sclerotic foci of the femur

**Synonyms**
Femoral slcerotic foci

**Definition (free text, please give PubMed ID)**
Lesions within the femur that demonstrate increased bone density. PMID30430046

**Parent term (use hpo.jax.org/app)**
#10451 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
BUSCHKE-OLLENDORFF SYNDROME, isolated osteopoikilosis (omim166700) PMID30430046

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000943
- label: Femoral osteopoikilosis
- parents: [Abnormal femur morphology (HP:0002823)](https://hpo.jax.org/app/browse/term/HP:0002823)
- pmid: PMID:30430046|PMID:30430046
- def: Multiple multiple, small, well-defined radiodense foci of cortical-like bone in the medullary space parallel to the trabeculae located in the femur. The loci correspond to condensations of the lamellar trabeculae on histology.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10469
- synonyms: Sclerotic foci of the femur(NONE)|Femoral slcerotic foci(NONE)
